common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -906,47 +905,79 @@ div.problem {
 // +Problem - Actions
 // ====================
 div.problem .action {
-  margin-top: $baseline;
+  @include margin($baseline 0 $baseline 0);
+  min-height: $baseline;
 
-  .save, .check, .show, .reset, .hint-button {
-    @include margin-right($baseline/2);
-    margin-bottom: ($baseline/2);
-    height: ($baseline*2);
-    vertical-align: middle;
-    text-transform: uppercase;
-    font-weight: 600;
-    @media print {
-      display: none;
+  // override default button hover
+  button {
+    &:hover {
+      background-image: none;
+      box-shadow: none;
+    }
+    &:focus {
+      box-shadow: none;
+    }
+    &.btn-default {
+      background-color: transparent;
+    }
+    &.btn-brand {
+      &:hover {
+        background-color: #0ea6ec;
+      }
+      &.check {
+        &.is-disabled, &.disabled {
+          background: transparent;
+          border-color: #d2d0d0;
+          color: #6b6969;
+        }
+      }
     }
   }
 
-  .save {
-    @extend .blue-button !optional;
+  @media (min-width: 768px) {
```

Comment:
@alisan617 and @chris-mike I understand moving the buttons above Submit for smaller screens. But it looks very cramped and inelegant-- we need to make sure this will look good on mobile as well. Is there a way to do better padding in this case?

![image](https://cloud.githubusercontent.com/assets/484484/17234704/4d85df52-5509-11e6-9698-e58dcece7141.png)